packages/main/cypress/specs/ResponsivePopover.cy.tsx

```diff
@@ -0,0 +1,122 @@
+import Button from "../../src/Button.js";
+import ResponsivePopover from "../../src/ResponsivePopover.js";
+
+describe("ResponsivePopover general interaction", () => {
+	it("header and footer are displayed by default", () => {
+		cy.mount(
+			<>
+				<Button id="btnOpen" />
+				<ResponsivePopover id="respPopover" opener={"btnOpen"}>
+					<div slot="header" />
+					<div slot="footer" />
+				</ResponsivePopover>
+			</>
+		);
+
+		cy.get("#respPopover").invoke("attr", "open", true);
+		cy.get("#respPopover").should("be.visible");
+
+		cy.get("#respPopover")
+			.shadow()
+			.find(".ui5-popup-header-root")
+			.should("exist");
+
+		cy.get("#respPopover")
+			.shadow()
+			.find(".ui5-popup-footer-root")
+			.should("exist");
+	});
+
+	it("header and footer are hidden on desktop", () => {
+		cy.mount(
+			<>
+				<Button id="btnOpen3" />
+				<ResponsivePopover id="respPopover3" content-only-on-desktop opener={"btnOpen3"}>
+					<div slot="header" />
+				</ResponsivePopover>
+			</>
+		);
+
+		cy.get("#btnOpen3").realClick();
```

Comment:
do you need to click on the button since RP is programatically opened? same applies for the following tests